### PR TITLE
Remove English restriction for translationWords

### PR DESCRIPTION
**Pull request by PurpleGuitar**
This PR refactors the translationWords code to pass the current source text language down to the loading functions.  This allows BW to show translationWords content for languages other than English.

### CODE REVIEW

**Review by AnonymousWalker — APPROVED** [review]

File: src/js/database.js [hunk 1]
```diff
@@ -450,9 +452,9 @@ function DataManager(db, resourceDir, sourceDir) {
             }
         },
 
-        getAllWords: function (dict) {
+        getAllWords: function (language_id, dict) {
             var mythis = this;
-            var container = "en_" + dict + "_tw";
+            var container = language_id + "_" + dict + "_tw";
```
Comment (rbnswartz) [inline src/js/database.js:457]: Might be a good place to use interpolation.

File: src/js/database.js [hunk 2]
```diff
@@ -423,22 +423,24 @@ function DataManager(db, resourceDir, sourceDir) {
             if (source.resource_id === "obs") {
                 dict = "bible-obs";
             }
-            var container = "en_" + dict + "_tw";
+            var container = source.language_id + "_" + dict + "_tw";
+            console.log(container);
```
Comment (rbnswartz) [inline src/js/database.js:427]: Might want to remove this console.log

File: src/js/database.js [hunk 3]
```diff
@@ -423,22 +423,24 @@ function DataManager(db, resourceDir, sourceDir) {
...
             var list = this.parseYaml(container, "config.yml");
+            console.log(list);
```
Comment (rbnswartz) [inline src/js/database.js:429]: This one too

**Review by rbnswartz — APPROVED** [review]

File: src/js/database.js (hunk 1 above)
Comment (PurpleGuitar) [reply src/js/database.js:455]: Fixed.

File: src/js/projects.js [hunk 4]
```diff
@@ -282,7 +282,7 @@ function ProjectsManager(dataManager, configurator, reporter, git, migrator) {
 
             if (meta.type.id === "tw") {
                 var dict = meta.project.id;
-                frames = dataManager.getAllWords(dict);
+                frames = dataManager.getAllWords(source.language_id, dict);
```
Comment (AnonymousWalker) [inline src/js/projects.js:285]: @PurpleGuitar source was not defined in this context, we can update this in another PR

**Pull request merged**